packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -70,69 +71,73 @@
 .secondaryVariant {
   border-color: var(--ColorBorderSecondary);
   background-color: var(--ColorBackgroundSecondary);
-  color: $color-primary-base;
+  color: var(--ColorTextSecondary);
 
   &:hover {
     border-color: var(--ColorBorderSecondaryHover);
     background-color: var(--ColorBackgroundSecondaryHover);
-    color: $white;
+    color: var(--ColorTextSecondaryHover);
   }
 
   &:focus {
     border-color: var(--ColorBorderSecondaryFocus);
     background-color: var(--ColorBackgroundSecondaryFocus);
-    color: $color-primary-base;
+    color: var(--ColorTextSecondaryFocus);
   }
 
   &:disabled {
     border-color: var(--ColorBorderSecondaryDisabled);
     background-color: var(--ColorBackgroundSecondaryDisabled);
-    color: $color-primary-light;
+    color: var(--ColorTextSecondaryDisabled);
   }
 }
 
 .dangerVariant {
-  background-color: $color-danger-base;
+  background-color: var(--ColorBackgroundDanger);
 
   &:hover {
-    border-color: $color-danger-dark;
-    background-color: $color-danger-dark;
+    border-color: var(--ColorBorderDangerHover);
+    background-color: var(--ColorBackgroundDangerHover);
   }
 
   &:focus {
-    @include outline($focus-ring-danger);
+    @include outline(var(--ColorFocusDanger), var(--FocusOutlineWidth));
 
-    background-color: $color-danger-dark;
+    background-color: var(--ColorBackgroundDangerFocus);
   }
 
   &:disabled {
-    border-color: $color-danger-light;
-    background-color: $color-danger-light;
+    border-color: var(--ColorBorderDangerDisabled);
+    background-color: var(--ColorBackgroundDangerDisabled);
   }
 }
 
 .dismissVariant,
 .dismissInvertedVariant {
-  padding-block: ($spacing-xs-em * 0.5);
-  padding-inline: ($spacing-xs-em * 0.5);
+  padding-block: calc(var(--SpacePaddingBlockDismiss) * 0.5);
+  padding-inline: calc(var(--SpacePaddingInlineDismiss) * 0.5);
   border: 0;
-  background-color: transparent;
-  color: inherit;
-  line-height: 1;
+  background-color: var(--ColorBackgroundDismiss);
+  color: var(--ColorTextDismiss);
+  line-height: var(--FontLineHeightDismiss);
 
   &:hover {
-    border-color: transparent;
+    border-color: var(--ColorBorderDismissHover);
+
+    /* TODO: var(--ColorBackgroundDismissHover); */
     background-color: transparentize($white, 0.6);
```

Comment:
need to work through how to handle this, [sass:color transparentize](https://sass-lang.com/documentation/modules/color#transparentize) wont' work on a custom property I imagine as the goal is to compile to `rbga` unless we perhaps structure the variable in that format rather than hex? it all seems very complicated but I'll try and get a decent solution here

Reply:
This is tricky. My gut says we should probably create some general tokens for transparency that use `rgba` rather than a hex-converting function. We can avoid the sass issues altogether that way.

I can get you the proper `rgba` here if it's not apparent from the output.

Reply:
@edburyenegren-okta good thoughts, I agree that if these values should up in multiple places a global token with the right rbga values makes sense. I need to double check on IE11 support for that (see my note above about compiled tokens and variables).

For now, I'll get the computed value here and include it as a component theme variable so we can avoid the sass function operating on a custom property